### PR TITLE
[PR against rd/meshcat-vis] Observable map fix

### PR DESCRIPTION
**Pull request by tkoolen**

### CODE REVIEW

**Comment by tkoolen** [comment]
cc:@rdeits

File: src/visualization.jl [hunk 1]
```diff
@@ -166,9 +166,9 @@ function Base.open(controls::SimulationControls, window::Window)
     body!(window, render_default(controls))
 end
 
-function SimulationStatus(controls::SimulationControls{<:Widget{:button}})
-    terminate = map(!iszero, observe(controls.terminate))
-    pause = map(isodd, observe(controls.pause))
```
Comment (rdeits) [inline src/visualization.jl]: This doesn't look right. The extra `Observable{Bool}(false)` shouldn't be necessary, and we need to call `observe()` on both controls. I'm not sure why this isn't working on your system, but it's been fine for me. I'm probably just on a strange combination of package versions, so I'll try to work it out.

Comment (tkoolen) [reply src/visualization.jl]: It's https://github.com/JuliaGizmos/Observables.jl/pull/15.

File: src/visualization.jl [hunk 2]
```diff
@@ -135,9 +135,9 @@ end
 
 @deprecate CallbackSet(vis, state::MechanismState; max_fps = 60.) CallbackSet(vis, state; max_fps = max_fps)
 
-struct SimulationControls{W <: Widget}
-    terminate::W
-    pause::W
+struct SimulationControls
+    terminate::Widget{:button}
+    pause::Widget{:button}
```
Comment (tkoolen) [inline src/visualization.jl]: Shouldn't have included this, but I still think it's a good change.

**Pull request merged**